### DISCUSSION TITLE
energy shifts

### DISCUSSION THREAD

**Discussion by pabufager** [ACEsuit/mace#1101] [Q&A]
Hello, We are performing a fine-tuning process that involves training on a dataset containing different systems. The atomic energies were provided using the -E0s tag. After completing the training, we plotted the predicted energy (EMACE) versus the DFT energy (EDFT) for  configurations in the dataset. We observed  linear relationships of the form: EMACE = EDFT + A
with different A values for each system.

Could this behavior be a consequence of training on multiple systems simultaneously? Thank you very much.

**Comment by gabor1** [comment]
No, this is some problem with data and/or input parameters. Please post your plots, energies per atom (directly as predicted by DFT and MACE), and also plot the predicted energies per atom with subtracting the isolated atom energies that you think should be appropriate. do this for both the training set and for test set (maybe colour points according to which systems)

**Reply by gabor1** [reply]
Note that when you fine-tune, if you are changing functional and pseudopotential, the atomisation energy is slightly different than the original model, so it takes some epochs for the model to readjust. But even the initial state (before any fine-tuning) shouldn't have a very large "A" (notation of your equation)

**Reply by pabufager** [reply]
Thank you for your answer, here is the picture 

<img width="1010" height="964" alt="Figure-foro" src="https://github.com/user-attachments/assets/262d0d83-12d3-42ef-89c8-13fb2df8a7d7" />

**Reply by gabor1** [reply]
these shifts are on the order of 1 meV/atom. my bet is that your DFT settings are not converged, and that lack of convergence varies depending on which "system" (colour in your plot) you are considering.

**Reply by gabor1** [reply]
it's actually quite hard to converge DFT to better than 1 meV/atom, you need a plane wave cutoff (if you are using plane waves) or integration grids (if you are using a local orbital code) that are quite extreme. in normal DFT work you don't need this because you have cancellations of error when you compute energy differences using the same unit cell

**Reply by gabor1** [reply]
also k-point convergence (assuming your system is periodic) is very slow in this regime.

**Reply by pabufager** [reply]
I agree. In any case, running MD simulations with the trained model should be reasonable, right?

**Reply by gabor1** [reply]
yes I think so. a potential that is 1 meV/atom accurate is very very good for most tasks (except maybe organic crystal structure prediction, where you need 5-10x more accuracy).

**Comment by gabor1** [comment]
make sure you use the "free energy" from the electronic structure calculation, that is the one that's consistent with the forces.

**Comment by pabufager** [comment]
Dear @gabor1,  I would like to further improve our model for the second generation (MPG2).
For the first generation (MPG1), we selected random structures obtained from MD simulations with MPA. This dataset (Input Set 1) was used for fine-tuning. Afterwards, we ran additional MD simulations using MPG1 and selected a new dataset (Input Set 2).

Do you have any advice on how we should proceed with the second-generation training? Thank you very much, Best, Paula

**Reply by gabor1sg** [reply]
I would pool your first and second generation data sets together, and use them in multihead fine-tuning
